merchantapi/src/main/java/fi/hel/verkkokauppa/merchant/model/ConfigurationModel.java

```diff
@@ -0,0 +1,21 @@
+package fi.hel.verkkokauppa.merchant.model;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import lombok.Data;
+import lombok.ToString;
+import org.springframework.data.elasticsearch.annotations.Field;
+import org.springframework.data.elasticsearch.annotations.FieldType;
+
+@Data
+@ToString
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class ConfigurationModel {
+    @Field(type = FieldType.Text)
+    public String key;
```

Comment:
Should these fields be set as private for field access control? Lomboks @DaTa -annotation will create necessary getters and setters for the values.

Reply:
That should be done but it differs from "common" project style so I will leave these as public fields even we have the getters and setters.

Reply:
Yes, we can mitigate it later in a general refactor ticket https://helsinkisolutionoffice.atlassian.net/browse/KYV-658 .